shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -308,31 +308,39 @@ - (void)removeSplashScreenView:(dispatch_block_t _Nullable)onComplete {
 }
 
 - (void)installFirstFrameCallback {
-  auto weak_platform_view = [_engine.get() platformView];
+  fml::WeakPtr<flutter::PlatformViewIOS> weak_platform_view = [_engine.get() platformView];
```

Comment:
Since at this point you are pretty much rewriting the whole method, could you fix the fact that all of the local variable names violate ObjC style? This should be `weakPlatformView`, and similarly for all the others.

Reply:
Done